fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalTVFRelation.java

```diff
@@ -107,7 +107,7 @@ public String toString() {
     public List<Slot> computeOutput() {
         return function.getTable().getBaseSchema()
                 .stream()
-                .map(col -> SlotReference.fromColumn(col, ImmutableList.of()))
+                .map(col -> SlotReference.fromColumn(col, ImmutableList.of(), null))
```

Comment:
This is different from computeOutput() in LogicalTVFRelation, which pass 'this' as third param. Is this expected?

Reply:
fixed